lips/lip-10.md

```diff
@@ -0,0 +1,231 @@
+---
+lip: 10
+title: Libra ID Spec
+authors: Andrey Chursin(@andll), Kevin Hurley (@kphfb)
+status: Draft
+type: Standard
+created: 11/03/2020
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Summary
+
+This LIP describes Libra ID - the human-readable identifier for user accounts.   
+
+In addition to describing the format of the Libra ID, this LIP also describes the off-chain protocol for the peer-2-peer payment with Libra ID.
```

Comment:
What is the main scenario that requires the off-chain p2p payment protocol?
For an amount of more than 1000, we must use TR metadata and signature. 
Is this a "nice-to-have" protocol for payments? any enforcement for that to take place before the actual transaction submission?

Reply:
With Libra ID in current form any p2p payment requires off-chain first, since there is no subadresses when Libra ID is used

We were considering having a shortcut protocol (for example including encrypted libra id in the metadata) that would not require off-chain communication. But so far it feels like it does not simplify things that much(since the off-chain API for libra id is pretty simple POST request).